features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/mapper/TextViewMapper.kt

```diff
@@ -69,10 +58,17 @@ open class TextViewMapper internal constructor(
             density
         )
 
+        val testValueObfuscationRule = when (mappingContext.privacy) {
```

Comment:
shouldn't be `textValueObfuscationRule` ?

Reply:
Also the one thing we need to consider here: based on my previous metrics, the text value obfuscation part takes quite a considerable time on the UI Thread (especially if the text is big enough). I would add here a follow - up task where we could try to move this operation in the Worker Thread following the same approach we have in place for resolving the Bitmaps from the actual elements.